libfabric.spec

```diff
@@ -17,6 +17,10 @@ Release:        1%{?dist}
 %{lua:
     rpm.define("dl_version " .. string.gsub(rpm.expand("%{version}"), "~", ""))
 }
+# this is a temporary fix to work around a URL issue
+%{lua:
+    rpm.define("dl_version_fix " .. string.gsub(rpm.expand("%{version}"), "~", "."))
```

Comment:
Oh yuck.  Are they going to fix that upstream?

Reply:
yes this is fixed